### ISSUE TITLE
Load models directly from run_id

### ISSUE THREAD

**Issue by thibo73800** [Visual-Behavior/aloception#85]
Load models directly from run_id

**Comment by anhtu293** [comment]
@thibo73800 is this issue solved ?

**Comment by thibo73800** [comment]
Not at all @anhtu293  
It could be usefull

**Comment by thibo73800** [comment]
Yes, it was referring to loading from run_id ckpt without using load_training and the LitModel but directly using the model itself.
